src/commands.py

```diff
@@ -243,3 +251,27 @@ async def action(self, message, *args, **kwargs):
                 "description": chosen_opt,
                 "color": 53380}
         )
+
+class help(command_on_message):
+    """
+    $help [command]
+
+    Displays description of provided command. If no command is provided, displays all commands
+    """
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.commands_list = {}
+        for cmd in command_on_message.__subclasses__():
+            self.commands_list[cmd.__name__] = cmd.__doc__
```

Comment:
Make the doctext a `@property` of the command class and create an embed instead of a single message.